WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -951,6 +950,7 @@ extension WordPressAppDelegate {
         /// It's necessary to target `PostCategoriesViewController` a second time to "reset" the UI element's `tintColor` for use in the app's Site Settings screen.
         UIView.appearance(whenContainedInInstancesOf: [PostCategoriesViewController.self, WPSplitViewController.self]).tintColor = .primary
 
+        setupComponentsAppearance()
```

Comment:
Good question! Since I realized that we have to call this method when `AppColor.accentColor` is changed, one option was to just make 2 distinct calls [here](https://github.com/tadija/WordPress-iOS/blob/d6d6b6db98ef7e84beb6f05cbbe63773905ee54c/WordPress/Classes/ViewRelated/Me/App%20Settings/App%20Color/AppColorListView.swift#L55-L56), but I figured it was better to have one centralized call for customizing global appearance, so I moved the original call of this method to be within the existing `customizeAppearance` method.